lib/sentry.dart

```diff
@@ -161,7 +161,7 @@ class SentryClient {
 
   @visibleForTesting
   String get postUri =>
-      '${dsnUri.scheme}://${dsnUri.host}/api/$projectId/store/';
+      '${dsnUri.scheme}://${dsnUri.host}${dsnUri.port != 80 && dsnUri.port != 443 ? ':' + dsnUri.port.toString() : ''}/api/$projectId/store/';
```

Comment:
This is probably a good time to switch from the `=>` shorthand to a `{ }` block syntax, and make port a variable.